Yank/pipeline.py

```diff
@@ -1347,7 +1348,7 @@ def _setup_system(self, system_file_path, pack, alchemical_charge,
                 for i, mol_id in enumerate(molecule_ids):
                     if mol_id not in self._pos_cache:
                         self._pos_cache[mol_id] = utils.get_oe_mol_positions(
-                                utils.read_oe_molecule(self.molecules[mol_id]['filepath']))
+                                utils.load_oe_molecules(self.molecules[mol_id]['filepath'], molecule_idx=0))
```

Comment:
Before this step, we separate multi-molecules files into several files containing a single molecule according to the `select` options (that defaults to `all`). So when we get to this point, there is only 1 molecule in the file we're reading in both cases.

I'll work on the docs update tonight on the flight!